crates/core/src/db/datastore/traits.rs

```diff
@@ -482,23 +482,19 @@ pub trait MutTxDatastore: TxDatastore + MutTx {
         table_id: TableId,
         relation: impl IntoIterator<Item = ProductValue>,
     ) -> u32;
-    /// Inserts `row` into the table identified by `table_id`.
+    /// Inserts `row`, encoded in BSATN, into the table identified by `table_id`.
     ///
-    /// Returns the generated column values (the [`AlgebraicValue`])
+    /// Returns the list of columns where values were replaced with generated ones
```

Comment:
```suggestion
    /// Returns the list of columns where values that were replaced with generated ones
```

Reply:
This doesn't scan. "[...] where values that were [...]" is incorrect.